src/main/java/seedu/travel/ui/ChartCard.java

```diff
@@ -151,7 +151,7 @@ private void createPieChart(List<RatingChart> ratingList) {
         for (int i = 0; i < tempRatingList.size(); i++) {
             RatingChart rating = tempRatingList.get(i);
             String ratingName = rating.getChartRating().toString();
-            if (i == 0) {
+            if (ratingName.equals("1")) {
```

Comment:
If possible, could you change the "1" into a static variable to avoid magic numbers?